### PR TITLE
RHSSO-1197 for 7.2 release

### PR DESCRIPTION
**Pull request by matthewhelmke**
These are changes for 7.2, as requested in https://issues.jboss.org/browse/RHSSO-1197?focusedCommentId=13497734&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13497734

### CODE REVIEW

**Comment by stianst** [comment]
@pskopek can you check this please?

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 1]
```diff
@@ -163,13 +163,13 @@ $ sudo subscription-manager repos --enable=jb-eap-6-for-rhel-<RHEL_VERSION>-serv
 Install the EAP 6 adapters for OIDC using the following command:
 
 ----
-$ sudo yum install eap6-keycloak-adapter
+$ sudo yum install eap6-keycloak-adapter-sso7_2-eap6
```
Comment (stianst) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: Should be keycloak-adapter-sso7_2

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 2]
```diff
@@ -163,13 +163,13 @@ $ sudo subscription-manager repos --enable=jb-eap-6-for-rhel-<RHEL_VERSION>-serv
...
 ----
 
 Install the EAP 6 adapters for SAML using the following command:
 
 ----
-$ sudo yum install eap6-keycloak-saml-adapter
+$ sudo yum install keycloak-saml-adapter-sso7_2-eap6
```
Comment (stianst) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc:172]: Should be keycloak-adapter-sso7_2

**Review by stianst — DISMISSED** [review]

File: securing_apps/topics/oidc/java/jboss-adapter.adoc (hunk 1 above)
Comment (matthewhelmke) [reply securing_apps/topics/oidc/java/jboss-adapter.adoc]: I just noticed the start of the adapter name still included "eap6-" here, but not below. Should it also be removed here?

File: securing_apps/topics/oidc/java/jboss-adapter.adoc (hunk 2 above)
Comment (matthewhelmke) [reply securing_apps/topics/oidc/java/jboss-adapter.adoc:172]: 👍

File: securing_apps/topics/oidc/java/jboss-adapter.adoc (hunk 1 above)
Comment (pskopek) [reply securing_apps/topics/oidc/java/jboss-adapter.adoc]: @matthewhelmke correct name is keycloak-adapter-sso7_2-eap6

File: securing_apps/topics/oidc/java/jboss-adapter.adoc (hunk 2 above)
Comment (pskopek) [reply securing_apps/topics/oidc/java/jboss-adapter.adoc:172]: @matthewhelmke this one is correct

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 3]
```diff
@@ -103,13 +103,13 @@ $ sudo subscription-manager repos --enable=jb-eap-7-for-rhel-<RHEL_VERSION>-serv
 Install the EAP 7 adapters for OIDC using the following command:
 
 ----
-$ sudo yum install eap7-keycloak-adapter
+$ sudo yum install keycloak-adapter-sso7_2
 ----
```
Comment (pskopek) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: correct adapter package name is eap7-keycloak-adapter-sso7_2

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 4]
```diff
@@ -103,13 +103,13 @@ $ sudo subscription-manager repos --enable=jb-eap-7-for-rhel-<RHEL_VERSION>-serv
...
 
 Install the EAP 7 adapters for SAML using the following command:
 
 ----
-$ sudo yum install eap7-keycloak-saml-adapter
+$ sudo yum install keycloak-saml-adapter-sso7_2
```
Comment (pskopek) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: correct adapter package name is eap7-keycloak-saml-adapter-sso7_2

File: securing_apps/topics/oidc/java/jboss-adapter.adoc (hunk 1 above)
Comment (pskopek) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: correct adapter package name is keycloak-adapter-sso7_2-eap6

**Review by pskopek — CHANGES_REQUESTED** [review]

**Review by pskopek — APPROVED** [review]

**Pull request merged**